contributors/devel/release/releasenotes.md

```diff
@@ -0,0 +1,94 @@
+# Creating PRs for release notes
+
+This document describes how to create PRs in Kubernetes repos so that the changes can be tracked with the
+Kubernetes release and published in release notes.
+
+## Curated release notes
+
+The release notes for each Kubernetes release are divided into several sections.
+
+All sections are grouped into subsections by component.
```

Comment:
@pwittrock the problem is I don't really know what a component is, I took various stabs in a further review comment, but I don't know if any of those hit the mark?